accounts/metric-stores/patch-stackdriver.yml

```diff
@@ -0,0 +1,16 @@
+#-----------------------------------------------------------------------------------------------------------------
+# Example configuration for enabling Google Stackdriver metrics
+#-----------------------------------------------------------------------------------------------------------------
+apiVersion: spinnaker.armory.io/v1alpha2
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  spinnakerConfig:
+    config:
+      metricStores:
+        stackdriver:
+          enabled: enabled
```

Comment:
Should be `true`?

Reply:
Whoops. Yes it should be `true`. I've pushed the fix.